src/utils/helper.js

```diff
@@ -477,12 +477,22 @@ export const updateInternalQuery = (
 			...defaultQueryOptions,
 			...(queryOptions || defaultQueryToExecute),
 		});
-		props.updateQuery({
-			componentId,
-			query,
-			value,
-			...queryParams,
-		});
+		// Certain components like ReactiveChart sends value as an object(array not included)
+		if (typeof value === 'object' && !Array.isArray(value)) {
```

Comment:
@SavvyShah This code would break the components which set value as an object and doesn't specify the `value` property. I would recommend handling it in `ReactiveChart` itself.